neon_utils/variables_override.py

```diff
@@ -0,0 +1,9 @@
+
+configuration_available = {'ttsVoice': {
```

Comment:
Is there a reason this is English only?

As a note, voices will probably be removed from config since they are TTS engine-specific and should be determined via API or something.

Reply:
No, there no particular reason other than this configuration is used to generate speaker data dict in custom-conversations. The default STT engines of Mycroft support only English (American and British, so the others in configuration_available are redundant).
Does this make sense to keep this config for compatibility with Neon?